src/ir/op.cc

```diff
@@ -122,6 +124,18 @@ TVM_REGISTER_GLOBAL("ir.RegisterOpAttr")
       }
     });
 
+TVM_REGISTER_GLOBAL("ir.RegisterOpLowerIntrinsic")
+    .set_body_typed([](String name, PackedFunc f, String target, int plevel, int can_override) {
+      if (Op::HasAttrMap(target + ".FLowerIntrinsic") &&
+          OpRegistry::Global()->Get(name) != nullptr &&
+          Op::GetAttrMap<FLowerIntrinsic>(target + ".FLowerIntrinsic").count(Op::Get(name))) {
+        ICHECK(can_override) << "Op " << name << "'s intrinsic lowering function " << target
```

Comment:
Use `CHECK` here because it is a user-facing error message.

```suggestion
        CHECK(can_override) << "Op " << name << "'s intrinsic lowering function " << target
```